Update _index.md (#33068)
DataDog/documentation: content/en/notebooks/advanced_analysis/_index.md

</div>
{{% /site-region %}}

{{< callout url="https://www.datadoghq.com/product-preview/additional-advanced-querying-data-sources/" header="Advanced Data Sources">}}

> Want to query a data source is not available? Use this form to request access.

Reviewer: ```suggestion
If you want to query data sources not yet available, use this form to submit your request.
```
